### ISSUE TITLE
Hide Password when loging in

### ISSUE THREAD

**Issue by p1ho** [mjkaufer/Messer#123]
## Summary

When I logged in, the password is out in the open, it'd be nice if the password stays hidden.

## Motivation

Security

**Comment by tomquirk** [comment]
Hi @p1ho , how do you mean "the password is out in the open"? As you enter the password, it should be hidden. Can you provide a screenshot?

**Comment by p1ho** [comment]
Yes for sure. I'm running Git Bash on Windows 10 with Node v10.15.3 and npm 6.4.1
![image](https://user-images.githubusercontent.com/22629067/58363948-82f95900-7e7a-11e9-8366-134e9e10b705.png)

**Comment by tomquirk** [comment]
huh, thats really interesting! It is hidden on MacOS and Linux afaik. We're simply using [this library](https://github.com/flatiron/prompt). I'll look into it a bit more though

**Comment by p1ho** [comment]
I see, I took a look at open issues and there were a couple that could be related:
 - [187](https://github.com/flatiron/prompt/issues/187)
 - [160](https://github.com/flatiron/prompt/issues/160)

**Comment by p1ho** [comment]
So I found the [relevant code](https://github.com/mjkaufer/Messer/blob/master/src/util/helpers.js#L28-L47) in the code base and tried to test it in isolation. Interestingly it works again...
This was the test code, the only dependency was [prompt](https://www.npmjs.com/package/prompt)
```Javascript
const prompt = require('prompt');

var x = new Promise((resolve, reject) => {
  prompt.get(
      [
        {
          name: "email",
          required: true,
        },
        {
          name: "password",
          required: true,
          hidden: true,
        },
      ],
      (err, result) => {
        if (err) return reject(err);
        return resolve(result);
      },
    );
  }).then(result => {
    console.log(result);
  });

```
and this was the output
![image](https://user-images.githubusercontent.com/22629067/58367066-59efbd00-7ea8-11e9-9a8a-f4c3c56f2f00.png)

It's also not version issue, because prompt should still be at version 1.0.0
